### DISCUSSION TITLE
remix & react spectrum

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [remix-run/remix#6267] [Q&A] [answered]
According to this link [https://remix.run/docs/en/main/guides/styling#css-side-effect-imports](https://remix.run/docs/en/main/guides/styling#css-side-effect-imports) remix is not loading react spectrum's css files. There is no error but remix just does not load css files. Is there any extra step? Please help.

**Comment by markdalgleish** [comment, marked as answer]
Have you set up CSS bundling? https://remix.run/docs/en/main/guides/styling#css-bundling. The styles from React Spectrum will be injected into the generated CSS bundle.

**Reply by markdalgleish** [reply]
To help avoid this issue in the future, all official Remix templates and stacks have been updated to include the CSS bundling setup out of the box.
